modules/Core/src/main/java/org/terasology/core/world/generator/ClimateSimulatorData.java

```diff
@@ -0,0 +1,131 @@
+package org.terasology.core.world.generator;
```

Comment:
Mind adding our project header to new files? It is included with our IntelliJ customizations as a template or you can just copy paste from any existing class and edit the year if needed